interface-elements-for-web/articles/diagram/first-look-at-the-diagram.md

```diff
@@ -0,0 +1,56 @@
+---
+title: First Look at the Diagram
+author: Svetlana Nikulina
+---
+# First Look at the Diagram
+
+![Tools](../../images/diagram-tools.png)
+
+## Diagram Tools
+
+- The **History** toolbar allows you to undo and redo actions. On mobile devises, the toolbar also displays the **Show Toolbox** button that shows/hides the **Toolbox**.
+
+    ![Show Toolbox Button](../../images/diagram-history-and-toolbox-button.png)
+
+- The **Toolbox** allows you to add shapes to your chart. Select the **View** > **Settings** > **Show Toolbox** command to switch the **Toolbox** visibility. On mobile devises, the **Show Toolbox** button is additionally displayed in the **History** toolbar.
+
+- The **Properties** toolbar allows you to [customize shape and connector style settings](create-a-new-chart.md#customize-shape-and-connector-style-settings), change the [chart layout](chart-layout.md), and specify [page settings](page-and-view-settings.md). Click the **Properties** button to show and hide the toolbar.
+
+- The **View** toolbar contains commands related to the **Diagram**'s [view settings](page-and-view-settings.md).
+
+- The **Context Toolbox** allows you to [create a shape at the end of a connector](work-with-chart-elements.md#create-a-shape-via-the-context-toolbox) in one click.

```

Comment:
Again, I'm not sure how "the **Context Toolbox** allows you to" do something

Reply:
in one click?

Reply:
Updated:
You can use the Context Toolbox to create a shape at the end of a connector.